.cmake-format.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: 2021-2023, Collabora, Ltd.
```

Comment:
this should probably be Khronos MIT copyright.

Reply:
This is a copy and pasted config file. But that's why it's cc0, there's nothing meaningfully copyrightable in it, so yeah that can be changed.

Reply:
@rbessems what would be the precise wording?

Reply:
from issue 57:

```cpp
// Copyright 2023, The Khronos Group Inc.
//
// SPDX-License-Identifier: MIT
```